nextflow_schema.json

```diff
@@ -10,14 +10,27 @@
             "type": "object",
             "fa_icon": "fas fa-terminal",
             "description": "Define where the pipeline should find input data and save output data.",
-            "required": [
-                "study_accession",
-                "reads_accession",
-                "blast_reference_genomes_folder",
-                "bwamem2_reference_genomes_folder",
-                "outdir",
-                "human_phix_blast_index_name",
-                "human_phix_bwamem2_index_name"
+            "oneOf": [
+                {
+                    "description": "Required fields when samplesheet is not provided.",
+                    "required": [
+                      "study_accession",
+                      "reads_accession",
+                      "blast_reference_genomes_folder",
+                      "bwamem2_reference_genomes_folder",
+                      "outdir",
+                      "human_phix_blast_index_name",
```

Comment:
Would these two index names be required also in the other case?

Reply:
Yes, I think you are right